design-library/src/style.css

```diff
@@ -1,7 +1,7 @@
-@import "css/index.css";
-
 @config "../tailwind.compile.config.ts";
 
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+@import "css/index.css";
```

Comment:
I think we're missing a proper CI check if the build compiles. IIRC `@import`s need to be at the top of the file, otherwise it will fail. This file is only used when building for production. I'll quickly check, but need to set up the repo locally first after switching laptops 😅

Reply:
It actually still compiles but it doesn't include the component styles in the stylesheet, which I think means the import is not processed. When I move the import at the top it works again. @u12206050 Any particular reason why you did this? With the `@layer` directives the CSS will still be inserted at the right point in the CSS.